src/libponyrt/asio/kqueue.c

```diff
@@ -282,4 +282,36 @@ void pony_asio_event_unsubscribe(asio_event_t* ev)
   retry_loop(b);
 }
 
+void pony_asio_event_resubscribe(asio_event_t* ev, uint32_t flags)
+{
+  if((ev == NULL) ||
+    (ev->magic != ev) ||
```

Comment:
we aren't doing this magic check in the epoll version. not sure if its not needed here or if it is needed in the epoll version or if all if fine with either.